### DISCUSSION TITLE
polygon custom controls first point

### DISCUSSION THREAD

**Discussion by alexmelo23** [fabricjs/fabric.js#7349] [Q&A]
Hi,

I need to change color for the first point in the polygon, any Idea how to do it ? I am using custom controls

[https://jsfiddle.net/ckitisak/qv6y283p/](url)

**Comment by asturur** [comment]
If you got a notification in email of answer, i confused a tab with another and that answer was meant for a different question

**Comment by asturur** [comment]
In that linked fiddle the first control is already different.
What are you missing?

**Reply by alexmelo23** [reply]
Hi
After I click the edit polygon button, I want to mark the first point White different color

**Comment by alexmelo23** [comment]
Hi
After I click the edit polygon button, I want to mark the first point White different color
